imports/wasi_snapshot_preview1/testdata/gotip/wasi.go

```diff
@@ -259,3 +264,46 @@ func mainStdin() error {
 func mainStdout() {
 	os.Stdout.WriteString("test")
 }
+
+func mainLargeStdout() {
```

Comment:
this is the same test as [bug449.go](https://github.com/golang/go/blob/b88bd917b8dd8aabc471b1de41b3ee8c0d6eeabe/test/fixedbugs/bug449.go) except here it doesn't show the issue, because I cannot reproduce the same exact setup. I think the trick is that the test case in the Go repo is run via `exec` on a `wazero` binary, while here we are instantiating wazero without forking.

Reply:
resolved by actually forking